ui/src/components/user-details.tsx

```diff
@@ -234,6 +234,7 @@ export class UserDetails extends Component<UserDetailsProps, UserDetailsState> {
   }
 
   parseMessage(msg: WebSocketJsonResponse) {
+    console.log(msg);
```

Comment:
You probably want to remove this log and the one below.

Reply:
I have these console logs on all the websocket responses, but in a PR by someone else they were removed for some reason, which didn't let me see the responses. The main reason for them is, that I don't know how to view the results of websocket network calls, unlike an http req, where you can easily see the response.

Reply:
Hmm I dont know about ts/js, but wouldnt it be better to use some kind of logging library so that logging is disabled in production?

Reply:
Since its browser logs I need to see (even and especially for finding bugs in prod), it doesn't really make sense to set up production vs development logging. I might really need to see the browser logs from a user when debugging a problem.